logic/concludable/ConjunctionConcludable.java

```diff
@@ -19,22 +19,29 @@
 
 import grakn.common.collection.Pair;
 import grakn.core.common.exception.GraknException;
-import grakn.core.common.parameters.Label;
 import grakn.core.logic.Rule;
-import grakn.core.logic.Unification;
 import grakn.core.pattern.constraint.Constraint;
 import grakn.core.pattern.constraint.thing.HasConstraint;
 import grakn.core.pattern.constraint.thing.IsaConstraint;
 import grakn.core.pattern.constraint.thing.RelationConstraint;
+import grakn.core.pattern.constraint.thing.RelationConstraint.RolePlayer;
```

Comment:
This allows me to write RolePlayer instead of RelationConstraint.RolePlayer, which gets quite verbose. However, having `import X` and `import X.Y` together is perhaps less than desirable?